lib/internal/bootstrap/pre_execution.js

```diff
@@ -73,6 +73,10 @@ function prepareMainThreadExecution(expandArgv1 = false) {
   assert(!CJSLoader.hasLoadedAnyUserCJSModule);
   loadPreloadModules();
   initializeFrozenIntrinsics();
+
+  if (getOptionValue('--expose-primordials')) {
+    global.primordials = primordials;
```

Comment:
We may want to have this emit a warning whenever it is used (similar to the warning emitted when we pull `require('internal/test/binding')` in to a test. We need to be clear that this is not intended as a public API.